pkg/limayaml/default.yaml

```diff
@@ -264,7 +270,7 @@ hostResolver:
 # that has an IPv4 address, to the list. In case this still doesn't work (e.g. VPN
 # setups), the servers can be specified here explicitly. If nameservers are specified
 # here, then the configuration from network preferences will be ignored.
-# dns:
+dns:
```

Comment:
Why?

Reply:
Just to make it more visible when looking at the file with syntax highlighting. I thought other list properties were not commented out either, like `networks`, but now I see this is inconsistent, e.g. `provision`, `probes`, and `portForwards` are all commented out.

Personally I would prefer to uncomment all the top-level list properties, but it doesn't really belong into this PR. I will revert it, and then we can decide if we either want to comment out `networks`, or to uncomment them all.

Reply:
@AkihiroSuda Do you still want to split out the change, even with the additional comments in place? I think it is pretty clear now. And it's not like the example doesn't work; it should continue to work as before. It just won't have the new functionality.

I can split it still, if you prefer, but I really don't think this is necessary.